x11-misc/x2x/x2x-1.30.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+DESCRIPTION="A utility to connect the mouse and keyboard to another X"
+HOMEPAGE="https://github.com/dottedmag/x2x"
+SRC_URI="https://github.com/dottedmag/x2x/archive/refs/tags/debian/1.30-10.tar.gz"
```

Comment:
You should rename the tarball. Having a file named `1.30-10.tar.gz` is meaningless and also prone to name conflicts. Also note that you may want to use $PV here.